### ISSUE TITLE
"Set passwords to never expire" Enable and users configured to password expires

### ISSUE THREAD

**Issue by FernandoGualano** [MicrosoftDocs/microsoft-365-docs#10753]
Hi Team!
What happens if “Set passwords to never expire” option is Enabled but there are users who have the PasswordNeverExpires parameter set to $False? What would the behavior be?

Thanks!
Fernando

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d322ca91-2125-7486-887f-7c313f2fbcf9
* Version Independent ID: a139620d-3330-a687-709a-35bb270101e4
* Content: [Set the password expiration policy for your organization - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide)
* Content Source: [microsoft-365/admin/manage/set-password-expiration-policy.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/manage/set-password-expiration-policy.md)
* Service: **microsoft-365-business**
* GitHub Login: @KwekuA
* Microsoft Alias: **kwekua**

**Comment by msbemba** [comment]
@FernandoGualano Thank you for your feedback. The Password Policies take effect and it doesn't matter what the PasswordNeverExpires attribute is set to.

Hope this helps!

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @FernandoGualano Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by Shaleed** [comment]
Hello, I've tried the following case that includes the one in this issue. The result is not what I expected from what msbemba said.

1. Original situation:
   a. “Set passwords to never expire” enabled (by default)

2. What I changed:
   a. I disabled the “Set passwords to never expire” flag (to set the expiration to 90 days for all users);
   b. I set no expiration password through PowerShell to only 5 users (PasswordNeverExpires = True).
   c. Using the Get-AzureADUser command to get all users and related Password Expiration Policy status correctly gives all "PasswordNeverExpires = false" except for those 5 users.

3. What I tested:
   a. After all this is turned back on the “Set passwords to never expire” flag;
   b. And then I used the Get-AzureADUser command to get all users and related Password Expiration Policy status;
   c. The result was the same as step 2c.

I expected all PasswordNeverExpires to be set to true instead. So how does this work?
